lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -355,7 +355,7 @@ module.exports = class MethodMappings {
           };
         },
 
-        async locateMultipleElementsByElementId({id, using, value}) {
+        async locateMultipleElementsByElementId({id, _using, value}) {
```

Comment:
this shouldn't be changed.

Reply:
will remove, not a biggie

Reply:
This is important; these are not argument names, but object keys, so changing them will screw it up very badly. I see it's probably done automatically by some tool which (wrongly) looks at unused parameters, so please disable that.